src/main/java/Duke.java

```diff
@@ -1,10 +1,275 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String LINE = "-----------------------------";
+    private static final String FIRST_GREETING = "Hello from\n";
+    private static final String SECOND_GREETING = "Hello there! I'm Duke :D";
+    private static final String PROVIDE_SERVICE = "What can I do for you?";
+    private static final String GOODBYE = "Bye! Hope to see you soon :D";
+    private static final String BYE = "bye";
+    private static final String LIST = "list";
+    private static final String MARK = "mark";
+    private static final String MARKED = "Nice! I've marked this task as done:";
+    private static final String UNMARK = "unmark";
+    private static final String UNMARKED = "Nice! I've unmarked this task as undone:";
+    private static final String SPACEBAR = " ";
+    private static final String ADDED = "Got it. I have added this task: ";
+    private static final String NUMBER_OF_TASKS_FIRST_HALF = "Now you have ";
+    private static final String NUMBER_OF_TASKS_SECOND_HALF = " tasks in the list.";
+    private static final String INVALID_INDEX = "The index that you have indicated is invalid, please try again.";
+    private static final String TODO = "todo";
+    private static final String DEADLINE = "deadline";
+    private static final String BY = "/by";
+    private static final String EVENT = "event";
+    private static final String AT = "/at";
+    private static final int START_INDEX_OF_TODO = 5;
+    private static final int START_INDEX_OF_DEADLINE = 9;
+    private static final int START_INDEX_OF_TIMING = 4;
+    private static final int START_INDEX_OF_EVENT = 6;
+    private static final int LENGTH_OF_SPACE = 1;
+    private static final int MINIMUM_INDEX = 0;
```

Comment:
I think that it is good to make use of constants, but I think that it might be overwhelming to make values that are only being used once or twice a constant. Perhaps you could scan through the constants and decide which are really necessary?